components/input.css

```diff
@@ -0,0 +1,26 @@
+@import url(/stylesheets/main.css);
+
+.input {
+  display: block;
+  padding: 0.5rem;
+  width: 15rem;
+  margin-top: 5px;
+}
+
+.focus-input {
+  border: 2px solid hsl(var(--secondary-color));
+}
+
+.disabled-input {
+  pointer-events: none;
+}
+
+.error-input {
+  border: 2px solid hsl(var(--red-color));
+  background-color: hsl(var(--red-light), 0.1);
+}
+
+input[type="text"]:disabled{
+  background: hsla(0, 0%, 50%,0.1);
```

Comment:
do create a variable to keep consistency in the code, as this particular hsl sticks out

Reply:
will make the necessary changes.

Reply:
addressed the changes in latest commits